sklearn/utils/_get_params_html_repr.py

```diff
@@ -0,0 +1,106 @@
+# Authors: The scikit-learn developers
+# SPDX-License-Identifier: BSD-3-Clause
+
+import html
+import reprlib
+from collections import UserDict
+
+from sklearn._config import get_config
+
+
+class ReprHTMLMixin:
```

Comment:
OK, I think that having this file tells me that it is time to have a small refactoring. Let's organize the file in this manner:

```
sklearn/
└── utils/
    └── _repr_html/
        ├── __init__.py
        ├── _base.py
        ├── _estimator.py
        └── _params.py
```

We need to move the `.js` and `.css` as well.

The idea is to create a private module `_repr_html` to have only utilities inside this module. The `ReprHTMLMixin` would be located in the `base.py` while the dedicated info for the estimator or parameter will be split in the other module.

Reply:
For future reference (to myself), the "_" not needed for `_base.py`, `_estimator`, etc. as shown in the tree above